doc/classes/PhysicsServer2D.xml

```diff
@@ -651,8 +651,9 @@
 			<param index="1" name="shape_idx" type="int" />
 			<param index="2" name="enable" type="bool" />
 			<param index="3" name="margin" type="float" />
+			<param index="4" name="direction" type="Vector2" />
 			<description>
-				Sets the one-way collision properties of the body's shape with the given index. If [param enable] is [code]true[/code], the one-way collision direction given by the shape's local upward axis [code]body_get_shape_transform(body, shape_idx).y[/code] will be used to ignore collisions with the shape in the opposite direction, and to ensure depenetration of kinematic bodies happens in this direction.
+				Sets the one-way collision properties of the body's shape with the given index. If [param enable] is [code]true[/code], the one-way collision direction given by [param direction] will be used to ignore collisions with the shape in the opposite direction, and to ensure depenetration of kinematic bodies happens in this direction.
```

Comment:
The old bit is still relevant. If you rotate the body, the one-way direction will also get rotated.

Reply:
Very nastily written remark, though, may I add. The code tag makes it harder to read.